build

```diff
@@ -65,18 +65,18 @@ make_relup() {
     if [ -d "$releases_dir" ]; then
         while read -r zip; do
             local base_vsn
-            base_vsn="$(echo "$zip" | grep -oE "[0-9]+\.[0-9]+\.[0-9]+(-[0-9a-f]{8})?")"
+            base_vsn="$(echo "$zip" | grep -oE "[0-9]+\.[0-9]+\.[0-9]+(-[0-9a-f]{8})?" | head -1)"
```

Comment:
We use a while loop to read the zip file, so do we still need `head -1`?

Reply:
Assume the filename is `emqx-ee-4.4.0-6d388f8b-otp23.2.5-macos10-amd64.zip`, `head -1` makes sure the value of `base_vsn` is `"4.4.0-6d388f8b"` not `"4.4.0-6d388f8b\n23.2.5"`.

Reply:
OK, Cool